README.md

```diff
@@ -50,3 +51,29 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+
+
+### Answer
+
+#### The total worst-case time complexity is constant with O(len^2), len is the number of vertices.
+
+1. The function twoOptSwap takes the route length O(len), then createRoute.slice(i+1, k+1) is create a new array with time complexity of O(k-i+1), and then reverses elements in place. So, the worst-case time complexity is O(len).
+
+2. The countPathLength function iterating over route and count to totalLength whose worst-case time complexity is O(len)
+
+3. tsp_ls takes O(len^2)
+
+So, the total complexity of this implementation is O(len^2).
```

Comment:
Don't follow you here -- the loop runs $n^2$ times, but what happens inside the loop?

Reply:
This comment is still relevant.